README.md

```diff
@@ -180,6 +181,7 @@ To run the example project, clone the repo, and run `pod install` from the Examp
 
 ## Installation
 
+### CocoaPods
```

Comment:
Very minor. Let's put a blank line underneath so the spacing is consistent between the "Installation/CocoaPods" and "Installation/Carthage" sections.

Reply:
Fixed!